tests/github_api/prs_test.py

```diff
@@ -103,3 +124,33 @@ def get_is_mergeable_side_effect(api, urn, pr_num):
         ready_prs_list = [pr for pr in ready_prs]
         self.assertTrue(len(ready_prs_list) is 1)
         self.assertTrue(ready_prs_list[0].get("number") is 11)
+
+
+    @patch("github_api.prs.get_events")
+    def test_get_pr_last_updated_with_events(self, mock_get_events):
+        mock_get_events.return_value = \
+            create_mock_events([("PushEvent", "test_ref", "2017-01-01T00:00:10Z"),
+                                ("PushEvent", "blah", "2017-01-03T00:00:10Z")])
+
+        api = MagicMock()
+        api.BASE_URL = "api_base_url"
+
+        last_updated = prs.get_pr_last_updated(api,
+                create_mock_pr(10, "OK", "2017-01-02T00:00:00Z"))
+
+        self.assertEquals(last_updated, arrow.get("2017-01-01T00:00:10Z"))
```

Comment:
`assertEquals` is deprecated in favor of `assertEqual` – small change but would get rid of the warning in the tests. Also on line 156